### PR TITLE
Change Appearance format to v3.2-preview.2

### PR DESCRIPTION
**Pull request by shijiayuan**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json [hunk 1]
```diff
@@ -347,22 +347,36 @@
...
       "type": "object",
       "required": [
-        "style",
-        "styleConfidence"
+        "style"
       ],
       "properties": {
         "style": {
-          "description": "The text line style.",
+          "description": "An object representing the Style of the text line.",
```
Comment (bojunehsu) [inline specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json:357]: Recommend lower case "style".

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json [hunk 2]
```diff
@@ -347,22 +347,36 @@
...
+    "Style": {
+      "description": "An object representing the style of the text line.",
+      "type": "object",
+      "required": [
+        "name",
+        "confidence"
+      ],
+      "properties": {
+        "name": {
+          "description": "The text line style name.",
```
Comment (bojunehsu) [inline specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json:372]: Recommend listing the currently supported values in the description with indication that additional styles may be added in the future.

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json [hunk 3]
```diff
@@ -347,22 +347,36 @@
...
           "type": "string",
           "enum": [
```
Comment (bojunehsu) [inline specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json:374]: Need to add x-ms-enum.modelAsString=true.  See https://github.com/Azure/azure-rest-api-specs/blob/aa4bbdafcabcf4726a8ccbe2ab529cb4fe48ad6f/specification/cognitiveservices/data-plane/ComputerVision/preview/v3.1-preview.2/Ocr.json#L441 for an example.

http://azure.github.io/autorest/extensions/#x-ms-enum

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json [hunk 4]
```diff
@@ -0,0 +1,500 @@
+{
```
Comment (bojunehsu) [inline specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json:1]: Please check with @TFR258 to see if we want to name this v3.2-preview.1 to anticipate the possibility of preview.2.

**Review by bojunehsu — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json (hunk 1 above)
Comment (shijiayuan) [reply specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json]: Thanks! revised

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json (hunk 2 above)
Comment (shijiayuan) [reply specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json]: Add options to description

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json (hunk 3 above)
Comment (shijiayuan) [reply specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json:378]: Thanks!

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json [hunk 5]
```diff
@@ -369,8 +369,12 @@
       ],
       "properties": {
         "name": {
-          "description": "The text line style name.",
+          "description": "The text line style name, including handwriting and other.",
           "type": "string",
+          "x-ms-enum": {
+            "name": "name",
```
Comment (bojunehsu) [inline specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json:375]: This name will be used as the default class/enum name when auto-generating the SDK.  Consider "TextStyle" instead.

**Review by bojunehsu — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json (hunk 5 above)
Comment (shijiayuan) [reply specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json:375]: In our json result of read/layout, we have 'name' as key, this is what we delivery for Layout 2.1-preview.2

Comment (shijiayuan) [reply specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json]: Thanks for reminder, I have updated the name.

**Comment by bojunehsu** [comment]
Please search and edit all version numbers in the Swagger and examples.  There are currently lots of inconsistencies.

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json (hunk 4 above)
Comment (shijiayuan) [reply specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json]: Thanks! Have checked with Thomas, change the name as v3.2-preview.2.

**Review by bojunehsu — APPROVED** [review]

**Review by johanste — APPROVED** [review]

**Pull request merged**